### PR TITLE
add ability to configure xms and xmx

### PR DESCRIPTION
**Pull request by fvanderbiest**
Untested for now ...

### CODE REVIEW

**Comment by landryb** [comment]
Myeah.. why restricting yourself to xmx/xms ? In my production playbooks i have something like:

```
      geonetwork: {
        port: 8380,
        control_port: 8305,
        javaver: java-7-openjdk-amd64,
        javaopts: [ "-Xms3G", "-Xmx3G", "-XX:MaxPermSize=512m",
          "-Dgeorchestra.datadir={{ georchestra_datadir }}",
          "-Dgeonetwork.jeeves.configuration.overrides.file={{ georchestra_datadir }}/geonetwork/config/config-overrides-georchestra.xml"
        ]
      }
```

and the template loops on javaopts:
```
{% for opt in item.value.javaopts %}
JAVA_OPTS="$JAVA_OPTS {{ opt }}"
{% endfor %}
```

**Comment by fvanderbiest** [comment]
Much better, indeed, thanks Landry for your feedback.

File: playbooks/georchestra.yml [hunk 1]
```diff
@@ -103,9 +103,24 @@
...
+        port: 8280,
+        control_port: 8205,
+        xms: 1G,
+        xmx: 2G
+      },
+      geoserver: {
+        port: 8380,
+        control_port: 8305,
+        xms: 1G,
+        xmx: 1G
```
Comment (fvanderbiest) [inline playbooks/georchestra.yml]: XMX values very low. 
I'm not sure this will allow newer geoserver & geonetwork versions to start.
Will try it.

Comment (fvanderbiest) [reply playbooks/georchestra.yml]: GeoNetwork is currently unhappy, but unrelated at first sight

Comment (fvanderbiest) [reply playbooks/georchestra.yml]: https://github.com/georchestra/georchestra/issues/2986

Comment (fvanderbiest) [reply playbooks/georchestra.yml]: Surprised to see it still runs fine with 4Gb !

**Pull request merged**